src/platforms/python/common/configuration/integrations/flask.mdx

```diff
@@ -0,0 +1,32 @@
+---
+title: Flask
+---
+
+The Flask integration adds support for the [Flask framework](https://flask.palletsprojects.com/en/1.1.x/). For more information, see our [Flask integration guide](/platforms/python/guides/flask/).
```

Comment:
```suggestion
The Flask integration adds support for the [Flask framework](https://palletsprojects.com/p/flask/). For more information, see our [Flask integration guide](/platforms/python/guides/flask/).
```

Link to more general project page, instead of docs page of a specific version.